README.md

```diff
@@ -0,0 +1,26 @@
+# OpenShift Builds Catalog
+This repository contains all the supported ClusterBuildStrategies with Builds for Red Hat OpenShift.
+Not sure what a ClusterBuildStrategy is? Let's start with "What is [Builds for Red Hat OpenShift](https://github.com/shipwright-io/operator/)".
```

Comment:
@jkhelil I have added this to set up some context for someone who has landed on the repo for the first time. I am inclined to replace the hyperlink with our downstream documentation, once GAed. Please let me know your thoughts.

Reply:
Lgtm